doc/source/tutorials/lab.rst

```diff
@@ -299,12 +342,52 @@ a two-qubit system:
                 threshold: 0.0002694329123116206
                 iq_angle: 4.912447775569025
 
+And in the case of having a chip with coupler qubits
+we need to following changes to the previous runcard:
+
+.. code-block::  yaml
+
+    qubits: [0, 1]
+    couplers: [0]
+
+    topology: {0: [0, 1]}
+
+    native_gates:
+        two_qubit:
+            0-1:
+                CZ:
+                - duration: 30
+                amplitude: 0.6025
+                shape: Rectangular()
+                qubit: 2
+                relative_start: 0
+                type: qf
```

Comment:
I am not sure but I think the indentation needs to be
```suggestion
                - duration: 30
                  amplitude: 0.6025
                  shape: Rectangular()
                  qubit: 2
                  relative_start: 0
                  type: qf
```
At least I got some issues when I try to use it as it is.